src/EnergyPlus/DaylightingManager.cc

```diff
@@ -5579,13 +5579,7 @@ namespace DaylightingManager {
                     }
                 }
                 if (!daylightControlFound) {
-                    ShowSevereError("DaylightingDevice:Tubular = " + TDDPipe(PipeNum).Name + ":  is not connected to a Zone that has Daylighting.  ");
-                    ShowContinueError("Add Daylighting:Controls one of the following zones in the same enclosure:  ");
-                    for (int const zoneNum2 : DataViewFactorInformation::ZoneSolarInfo(Surface(SurfNum).SolarEnclSurfIndex).ZoneNums) {
```

Comment:
Side note: This line actually caused the file without daylighting to crash in a develop debug build (array bounds violation), so this change resolves that. This should have been `.SolarEnclIndex`

Reply:
Good to know, thanks.